src/ARControllerNFT.ts

```diff
@@ -918,7 +918,7 @@ export default class ARControllerNFT implements AbstractARControllerNFT {
           q += 4;
         }
       } else if (this.grayscaleEnabled == true) {
-        this.videoLuma.set(this.grayscaleSource);
```

Comment:
i can't uinderstand why this will make fail, with an out bounds error. I will investigate.